apps/src/lab2/progress/ProgressContainer.tsx

```diff
@@ -8,7 +8,6 @@ import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
 import React, {useCallback, useEffect, useRef} from 'react';
 import {useSelector} from 'react-redux';
 import {setValidationState} from '../lab2Redux';
-import {ProjectLevelData} from '../types';
```

Comment:
Just confirming that this cleanup is finishing up the absorbing of fields in `level_data` back to `level_properties` as part of [this convo](https://codedotorg.slack.com/archives/C044AGTKW3D/p1690844666853189?thread_ts=1690842845.194209&cid=C044AGTKW3D).

Reply:
Yep! We had already started using the level_properties fields for localization purposes, so this only affects one level in the music-accessibility script as far as I can tell (which I will fix before merging).